src/posts/interning-at-khan-academy.md

```diff
@@ -0,0 +1,53 @@
+title: "Interning at Khan Academy: from student to intern"
+published_on: November 19, 2016
+author: Shadaj Laddad
+team: Web Frontend
+...
+
+This past summer, I had the amazing opportunity to work on a product I use as a student every day -- Khan Academy. During my internship, I worked on the web frontend team on the XOM (x-on-mobile) initiative to bring Khan Academy exercises to mobile devices. In this blog post, I’ll be going through a highly-compressed summary of my two months at Khan Academy, but since most of my work here was in open-source projects you can check out my work in detail on GitHub ([https://github.com/Khan/perseus/commits?author=shadaj](https://github.com/Khan/perseus/commits?author=shadaj), [https://github.com/Khan/math-input/commits?author=shadaj](https://github.com/Khan/math-input/commits?author=shadaj)).
```

Comment:
Style nits:

* Colon instead of dash in "every day: Khan Academy".
* Full URLs are pretty bulky as text. Maybe `(1, 2)` instead?

Stray thoughts:

* The last sentence is kinda bulky and can maybe be broken up a bit—heck, the GitHub links might make more sense in the conclusion or as a postscript instead, since the reader is pretty darn unlikely to be interested in consulting your commit history until they know what it's about :P
* I wonder if "x-on-mobile" is a sufficient explanation of the joke, but I can't easily find a concise way to do it, and a bulky explanation is just distracting… Maybe "the amusingly-named X-on-Mobile initiative (XOM)"?